.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+
+## Context
```

Comment:
I don't see where the `Motivation and Context` thingie is, but I can guess that `Context` means the context when the error happened "I had the browser open, with 50 other tabs, while running 3 vms...." and `Motivation and Context` is about why where you trying to do that, like "I hit the 'reboot' button because I had a process that was stuck but did not know how to stop it, all I wanted was to stop it.".